crates/uv-small-str/src/lib.rs

```diff
@@ -148,7 +148,7 @@ impl PartialOrd<SmallString> for rkyv::string::ArchivedString {
 #[cfg(feature = "schemars")]
 impl schemars::JsonSchema for SmallString {
     fn schema_name() -> Cow<'static, str> {
-        String::schema_name()
+        Cow::Borrowed("SmallString")
```

Comment:
These were intentionally `string`, what type we're using for this string internally is an implementation detail and should not be exposed.

Reply:
Changed to inline.

https://github.com/astral-sh/uv/pull/13693/commits/f2a71fe2d5f084bddab70516c0343d78cb9bf423